boxnav/boxsim.py

```diff
@@ -30,7 +32,6 @@ def simulate(args: Namespace) -> None:
 
     pbar_manager = enlighten.get_manager()
     actions_pbar = pbar_manager.counter(total=args.max_total_actions, desc="   Actions")
-    navigation_pbar = pbar_manager.counter(total=100, desc="Completion")
```

Comment:
Why did you remove the navigation percentage bar?

Reply:
I didn't think it was very helpful since boxsim would continue running based on the number of actions and not environment completion and it is used for data collection. It could be useful with smaller runs where you would be watching it the whole way through but not for when we were collecting the 100k image datasets for example or other large runs.